Moonshot/player/PlayerAudio.gd

```diff
@@ -0,0 +1,105 @@
+extends Node
+
+
+var running_sfx = load("res://player/assets/sounds/running.wav")
+onready var run_streamer = create_streamer(running_sfx)
+var sliding_sfx = load("res://player/assets/sounds/wallsliding.wav")
+onready var sliding_streamer = create_streamer(sliding_sfx)
+var jumping_sfx = load("res://player/assets/sounds/jump.wav")
+onready var jumping_streamer = create_streamer(jumping_sfx)
+var pain_sfx = load("res://player/assets/sounds/paingrunt.wav")
+onready var pain_streamer = create_streamer(pain_sfx)
+var death_sfx = load("res://player/assets/sounds/deathgrunt.wav")
+onready var death_streamer = create_streamer(death_sfx)
+var sizzle_sfx = load("res://player/assets/sounds/sizzle.wav")
+onready var sizzle_streamer = create_streamer(sizzle_sfx)
+var health_sfx = load("res://player/assets/sounds/healthpickup.wav")
+onready var health_streamer = create_streamer(health_sfx)
+var pickup_sfx = load("res://player/assets/sounds/game-pick-up-object.wav") 
+onready var pickup_streamer = create_streamer(pickup_sfx)
+
```

Comment:
Ok at the moment, but probably need a better way of organising these if we are gonna have any more sound effects.